pkg/cluster/rook_ceph.go

```diff
@@ -301,53 +301,53 @@ func (c *Controller) SetObjectStoreReplication(name string, level int, doFullRec
 	return nil
 }
 
-func (c *Controller) rookCephOperatorExec(cmd ...string) error {
-	rookOperatorLabels := map[string]string{"app": "rook-ceph-operator"}
+func (c *Controller) rookCephToolsExec(cmd ...string) error {
+	rookToolsLabels := map[string]string{"app": "rook-ceph-tools"}
```

Comment:
The label and container name needs to be parameterized so it will work for both 1.0.4 and 1.4.1